bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/DataClientVeneerApi.java

```diff
@@ -70,6 +72,10 @@ public class DataClientVeneerApi implements DataClientWrapper {
     this.clientOperationTimeouts = clientOperationTimeouts;
   }
 
+  interface paginatorFunction {
```

Comment:
Why lower case?
Also why do you need a separate interface for this? Can it be `Function<Query.QueryPaginator, ServerStream<Result>>` ?

Reply:
Also, why do we need a closure to begin with? why not pass the client directly?

Reply:
Didn't know about Function<...> :( Done this way.
Passing the client feels like it's giving the scanner too much power, when all I want it to do is call a specific function that given a paginator - gets the next page.

Reply:
Ended up passing the client, to be able to use the async API